### PR TITLE
Refactor to async

### PR DESCRIPTION
**Pull request by pgallik**

### CODE REVIEW

File: src/AddressRegistry.Api.BackOffice.Handlers.Sqs.Lambda/Requests/SqsLambdaAddressChangePostalCodeRequest.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+        public int AddressPersistentLocalId { get; set; }
+
+        /// <summary>
+        /// Map to ChangeAddressPostalCode command
+        /// </summary>
+        /// <returns>ChangeAddressPostalCode.</returns>
+        public ChangeAddressPostalCode ToCommand()
+        {
+            return new ChangeAddressPostalCode(
```
Comment (jvandaal) [inline src/AddressRegistry.Api.BackOffice.Handlers.Sqs.Lambda/Requests/SqsLambdaAddressChangePostalCodeRequest.cs:22]: I think ToCommand should have a PostalCode argument and be called like in the sync handler:
```
            var postInfoIdentifier = request.PostInfoId
                .AsIdentifier()
                .Map(x => x);
            var postalCode = new PostalCode(postInfoIdentifier.Value);

            var cmd = request.ToCommand(
                streetNamePersistentLocalId,
                postalCode,
                CreateFakeProvenance());
```

File: src/AddressRegistry.Api.BackOffice.Handlers.Sqs.Lambda/Requests/SqsLambdaAddressCorrectPostalCodeRequest.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+        public int AddressPersistentLocalId { get; set; }
+
+        /// <summary>
+        /// Map to CorrectAddressPostalCode command
+        /// </summary>
+        /// <returns>CorrectAddressPostalCode.</returns>
+        public CorrectAddressPostalCode ToCommand(MunicipalityId municipalityId)
+        {
+            return new CorrectAddressPostalCode(
```
Comment (jvandaal) [inline src/AddressRegistry.Api.BackOffice.Handlers.Sqs.Lambda/Requests/SqsLambdaAddressCorrectPostalCodeRequest.cs:22]: Same comment as with ChangePostalCode

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+## [3.50.4](https://github.com/informatievlaanderen/address-registry/compare/v3.50.3...v3.50.4) (2022-09-22)
```
Comment (ArneD) [inline CHANGELOG.md:1]: can you recheck what happend? In the main branch this is present and shouldn't be a diff. same with package.json

**Comment by ArneD** [comment]
superseded by #763

**Pull request closed without merging**